site/package.json

```diff
@@ -11,15 +11,18 @@
     "docs:build": "npm run docs:vuepress && npm run docs:sitemap",
     "docs:vuepress": "vuepress-vite build docs",
     "docs:fmt": "deno fmt",
-    "docs:sitemap": "sscli -b https://grammy.dev -r docs/.vuepress/dist --format xml --no-robots --no-clean"
+    "docs:sitemap": "sscli -b https://grammy.dev -r docs/.vuepress/dist --format xml --no-robots --no-clean",
+    "docs:serve": "serve docs/.vuepress/dist",
+    "deno:upgrade": "deno upgrade"
   },
   "keywords": [],
   "author": "KnorpelSenf",
   "license": "MIT",
   "devDependencies": {
-    "@vuepress/plugin-docsearch": "^2.0.0-beta.51",
+    "@vuepress/plugin-docsearch": "next",
```

Comment:
@roj1512 please confirm that you're comfortable with unpinned deps when reviewing

Reply:
I'm comfortable with `@vuepress/plugin-docsearch@next`, but we might not always be fine with `vuepress-vite@next`. Revert, @CikiMomogi?

Reply:
Ok